crispy_forms/tests/results/bootstrap4/test_layout/test_use_custom_control_in_select_true.html

```diff
@@ -1,7 +1,7 @@
 <form method="post">
     <div class="form-group" id="div_id_select"><label class=" requiredField" for="id_select">Select<span
                 class="asteriskField">*</span></label>
-        <div class=""><select class="custom-select form-control select" id="id_select" name="select">
```

Comment:
https://getbootstrap.com/docs/4.6/components/forms/#select-menu

This change is correct, we only need `custom-select` here.